templates/heat/config/00-default.conf

```diff
@@ -7,6 +7,8 @@ num_engine_workers=4
 auth_encryption_key={{ .AuthEncryptionKey }}
 use_stderr=true
 transport_url={{ .TransportURL }}
+# Keep the RPC call timeout in sync with HAProxy and Apache timeouts
```

Comment:
I don't think `rpc_response_timeout` default of `60` would work for heat. It's `600` in 17.1 along with apache/httpd timeout. Btw where is haproxy in the picture?

Reply:
it is the route timeout: https://docs.openshift.com/container-platform/4.17/networking/routes/route-configuration.html#nw-configuring-route-timeouts_route-configuration

Reply:
right, should we mention the route timeout instead of haproxy though it translates to that?

Reply:
I'll change it

Reply:
HAProxy as in the one used for OpenShift ingress. So we pass an annotation to the Route to configure HAProxy timeout on the OpenShift Route.

Does it make sense to group RPC with API? Feels like a loose relationship at best.

Reply:
from [OSPRH-10954](https://issues.redhat.com/browse/OSPRH-10954), I tried to follow what manila, cinder, glance did. And I saw that both manila and cinder grouped it:
- https://github.com/openstack-k8s-operators/manila-operator/blob/57ac502c086d0bb2092e124d3152ab9e31a4ad79/templates/manila/config/00-config.conf#L13
- https://github.com/openstack-k8s-operators/cinder-operator/blob/980b1ab26c714bbc0b7f4925517c68f8d3e53169/templates/cinder/config/00-global-defaults.conf#L32